src/Perl6/Pod.nqp

```diff
@@ -516,7 +648,21 @@ class Perl6::Pod {
 
         }
         return build_pod_strings(@strings);
+    }
+
+    our sub lines($S, :$no-chomp) {
+        # Takes a string and separates it into an array of lines at
+        # each newline.
+        #
+        # uses nqp::split(str $delimiter, str $string --> Mu)
+
+        # not usual:
+        return nqp::split("\n", $S) if $no-chomp;
```

Comment:
Does any consideration need to be given to `\r\n` line endings?

Reply:
Nick, I don’t know, and didn’t think of that.